### DISCUSSION TITLE
Can i use the upload files option through the perplexity API?

### DISCUSSION THREAD

**Discussion by waleedshoaib2** [perplexityai/api-discussion#90] [Q&A]
So the thing is my usecase involves uploading files that are over 128k tokens, so i need to upload them as files. It works through the interface but i didnt find any relevant docs related to it can you provide me with the documentation or tell me if its possible or not?

**Comment by jlane003** [comment]
Found this answered in another thread a few weeks ago, the answer is that it is not.  Here's the thread in question:  https://github.com/ppl-ai/api-discussion/discussions/76#discussion-7528518
